### PR TITLE
💬  Add featured WNFT

### PR DESCRIPTION
**Pull request by nwingt**
image
image

### CODE REVIEW

File: src/locales/zh-Hant.json [hunk 1]
```diff
@@ -305,15 +304,16 @@
...
+  "nft_featured_likenft1m990psscyulzgjyxk0k55yp07zt67lf68gdxxd3c2evdt4ujhr4qssv3ux_title": "謹此記下人生階段的小結 💭",
   "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "在這人人都可以成為創作者的年代，內容轉向數位，卻仍然沒有一個可以跨越平台限制，自由買賣字的地方。區塊勢以最多人訂閱的中文區塊鏈媒體的角度，紀錄了全球第一家可以閱讀自由，創作自主之地，請進入「無人書店」。",
   "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "歡迎來到全球第一家「無人書店」🔮",
-
   "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "「這週大部分時間身處加州海岸的紅樹林，容許我偷懶一下，長信欠奉，附上明信片一張，閒話日常⋯⋯」2022 年初夏，k 先生凌晨 5 點多坐在舊金山的偌大教堂的長椅，兩層樓高的圓拱大窗，逐漸透出晨光第一線，給遠方友人寫下了片言隻語。這帶有獨一無二郵戳的明信片，等著你去領取。",
   "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_title": "你有一封未讀的新郵件 📩",
+  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_description": "過去，現在，未來，似乎是大家描述時間的固有用字。光陰一去不復返，從無限延伸的時間線彼端回望當下，難免令身處現世的人感到焦慮和虛空。然而作者指出人類對時間的觀念不是自有永有的單向線性，而是受語言文化和社會發展等影響，可以是搖擺和循環、重底蘊多於數字的概念。到底我們對時間的理解從何而來，如何解決時間一直流逝所帶來的虛無感，讓作者從歷史長河中娓娓道來。",
+  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_title": "人類對時間的理解：影響、概念和迷思",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:316]: `  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_title": "人類對時間的理解：影響、概念和迷思 🕰️",`

File: src/locales/en.json [hunk 2]
```diff
@@ -314,21 +314,18 @@
...
-
   "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_title": "You have a new mail 📩",
+  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_description": "The past, present, and future are the everyday vocabulary with which all describe time. Time waits for no one, and when gazing back from the end of the infinite timeline, those living in the present cannot help but feel anxious and hollow. However, the author highlights that humanity's understanding of time is not an eternal, one-way linear concept but rather influenced by language, culture, and societal development and can be a concept of oscillation, cyclical in nature, and grounded more in essence than in numbers. From where does our comprehension of time arise, and how shall we overcome the emptiness that accompanies the constant passage of time? Let the author illuminate this perplexing enigma straight from the river of history.",
+  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_title": "Humanity's Understanding of Time: Influences, Concepts, and Enigmas",
```
Comment (williamchong) [inline src/locales/en.json:326]: `  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_title": "Humanity's Understanding of Time: Influences, Concepts, and Enigmas 🕰️",`

**Review by williamchong — CHANGES_REQUESTED** [review]
For some reason github won't let me propose change

File: src/locales/en.json (hunk 2 above)
Comment (nwingt) [inline src/locales/en.json:326]:
```suggestion
  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_title": "Humanity's Understanding of Time: Influences, Concepts, and Enigmas 🕰️",

```

File: src/locales/zh-Hant.json (hunk 1 above)
Comment (nwingt) [inline src/locales/zh-Hant.json:316]:
```suggestion
  "nft_featured_likenft1x7t6efwe4g5mnjzqjrrpy9yg8wqk9ktsytuyuy0u0dze09dc6gls7yuh22_title": "人類對時間的理解：影響、概念和迷思 🕰️",

```

**Review by williamchong — APPROVED** [review]

**Pull request merged**